### PR TITLE
Draining timeout docs

### LINKED ISSUE #1721: Draining timeout in launch config
Provide an option in group launch config to store draining timeout. Following can be the new launch config:
```
{
   "launchConfiguration":{
      "args":{
         "loadBalancers":[{"port": 80, "loadBalancerId": 237935}],
         "server":{
            "name":"autoscale_server",
            "imageRef":"7cf5ffc3-7b20-46fd-98e4-fefa9908d7e8"         
         },
        "draining_timeout": 30
      },
      "type":"launch_server"
   },
  ...
}
```
- [x] Implement in otter
- [x] Integration test
- [ ] Docs

### PR DESCRIPTION
**Pull request by manishtomar**
Closes #1721

### CODE REVIEW

**Comment by manishtomar** [comment]
@stanzikratel review please.

File: api-docs/rst/dev-guide/concepts.rst [hunk 1]
```diff
@@ -105,6 +105,12 @@ The launch configuration specifies the launch type along with server and load ba
                 if you use a load balancer so the load balancer can retrieve the IP address of new
                 servers. See :ref:`Load balancer parameters <load-balancers-parameters>`.
 
+        ``draining_timeout``
+             Specifies the number of seconds autoscale will put the CLB node in DRAINING
+             before deleting the node and eventually the server. This is used when scaling down.
```
Comment (stanzikratel) [inline api-docs/rst/dev-guide/concepts.rst]: just a minor change from autoscale to "Autoscale" for consistency. Also please add "mode" after DRAINING

File: api-docs/rst/dev-guide/concepts.rst [hunk 2]
```diff
@@ -105,6 +105,12 @@ The launch configuration specifies the launch type along with server and load ba
...
+             Not used when there is no ``loadbalancers`` configuration. Please note that
+             this feature only works with cloud load balancer.
```
Comment (stanzikratel) [inline api-docs/rst/dev-guide/concepts.rst]: Please add "a" before cloud load balancer.

File: api-docs/rst/dev-guide/api-operations/methods/put-update-launch-configuration-v1.0-tenantid-groups-groupid-launch.rst [hunk 3]
```diff
@@ -169,6 +169,11 @@ This table shows the body parameters for the request:
 |                               |             |Rackspace Support after they configure your load   |
 |                               |             |balancer pool.                                     |
 +-------------------------------+-------------+---------------------------------------------------+
+|args.\ **draining_timeout**    |Integer      |Number of seconds deleting server's associated     |
+|                               |*(Optional)* |cloud load balancer node will be put in DRAINING   |
+|                               |             |before deleting the node followed by the server.   |
+|                               |             |Must be between 30 and 3600 inclusive.             |
```
Comment (stanzikratel) [inline api-docs/rst/dev-guide/api-operations/methods/put-update-launch-configuration-v1.0-tenantid-groups-groupid-launch.rst]: This sentence is a little confusing. I would rewrite it like this: "Specifies the number of seconds for which the cloud load balancer node associated with the server that is being deleted will be put in DRAINING before the node is actually being deleted followed by the server. Must be between 30 and 3600 inclusive.

**Comment by stanzikratel** [comment]
@manishtomar I reviewed the draining docs and made some minor comments

**Pull request merged**